trade-finance-manager-api/src/v1/controllers/send-deal-submit-emails.js

```diff
@@ -148,7 +142,7 @@ const sendAinMinAcknowledgement = async (deal) => {
       // send a copy of the email to PIM
       const pimEmailResponse = await sendTfmEmail(templateId, pimEmail, emailVariables, deal);
       // send a copy of the email to the bank's general email address
-      const bankResponse = bankEmails.map((email) => sendTfmEmail(templateId, email, emailVariables, deal));
+      const bankResponse = await Promise.all(bankEmails.map((email) => sendTfmEmail(templateId, email, emailVariables, deal)));
```

Comment:
`bankResponses` 😛 

Not a blocker, can we change in the next PR please? Let's get this merged